code/WorkInProgress/MarqStuff.dm

```diff
@@ -349,6 +349,20 @@
 		O.setHeadMaterial(head_material)
 		O.setShaftMaterial(shaft_material)
 		return O
+
+	attackby(obj/item/W, mob/user, params)
+		. = ..()
```

Comment:
This should go after the other checks to avoid `Person hits the arrow with the quiver` messages